values.yaml

```diff
@@ -498,10 +498,10 @@ core:
   # Fill the name of a kubernetes secret if you want to use your own
   # TLS certificate and private key for token encryption/decryption.
   # The secret must contain keys named:
-  # "tls.crt" - the certificate
   # "tls.key" - the private key
-  # The default key pair will be used if it isn't set
   secretName: ""
+  # If not specifying a preexisting secret, a secret can be created from tokenKey and tokenCertificate and used instead. If neither secretName or tokenKey are specified, a certificate will be autogenerated.
```

Comment:
Please add a comment to emphasize the following two points, if using `tokenKey`:
* the content should be a multi-line string with one more indentation of `tokenKey`
* the content of `tokenKey` should be a `rsa` type privateKey

Reply:
@chaospuppy thank you for contributing to harbor-helm. Please help to update this PR to resolve the above comment.
Appreciate it!

Reply:
Updated comments and addressed @zyyw's error